diagnostics/model_accuracy/verify_inference_model.py

```diff
@@ -39,40 +39,17 @@ def download_repo(repo, target_name=None, version='master'):
     command_executor("git clone " + repo)
 
 
-def run_inference(model_name, models_dir):
-    parameters = \
-    '[{"model_type" : "Image Recognition", "model_name" : "Inception_v4", \
-        "cmd" : "OMP_NUM_THREADS=28 KMP_AFFINITY=granularity=fine,compact,1,0 \
-                python eval_image_classifier.py --alsologtostderr \
-                --checkpoint_path=/nfs/fm/disks/aipg_trained_dataset/ngraph_tensorflow/fully_trained/inception4_slim/inception_v4.ckpt \
-                --dataset_dir=/mnt/data/TF_ImageNet_latest/ --dataset_name=imagenet \
-                --dataset_split_name=validation --model_name=inception_v4"},\
-     {"model_type" : "Image Recognition", "model_name" : "MobileNet_v1", \
-        "cmd" : "OMP_NUM_THREADS=28 KMP_AFFINITY=granularity=fine,compact,1,0 \
-                python eval_image_classifier.py --alsologtostderr \
-                --checkpoint_path=/nfs/fm/disks/aipg_trained_dataset/ngraph_tensorflow/fully_trained/mobilenet_v1/mobilenet_v1_1.0_224.ckpt \
-                --dataset_dir=/mnt/data/TF_ImageNet_latest/ --dataset_name=imagenet \
-                --dataset_split_name=validation --model_name=mobilenet_v1"}, \
-     {"model_type" : "Image Recognition", "model_name" : "ResNet50_v1", \
-        "cmd" : "OMP_NUM_THREADS=28 KMP_AFFINITY=granularity=fine,compact,1,0 \
-                python eval_image_classifier.py --alsologtostderr \
-                --checkpoint_path=/nfs/fm/disks/aipg_trained_dataset/ngraph_tensorflow/fully_trained/resnet50_v1_slim/resnet_v1_50.ckpt \
-                --dataset_dir=/mnt/data/TF_ImageNet_latest/ --dataset_name=imagenet \
-                --dataset_split_name=validation --model_name=resnet_v1_50 --labels_offset=1"}, \
-     {"model_type" : "Object Detection", "model_name" : "SSD-MobileNet_v1", \
-        "cmd" : "OMP_NUM_THREADS=28 KMP_AFFINITY=granularity=fine,compact,1,0 \
-                python object_detection/model_main.py --logtostderr \
-                --pipeline_config_path=object_detection/samples/configs/ssd_mobilenet_v1_coco.config \
-                --checkpoint_dir=/nfs/site/disks/aipg_trained_dataset/ngraph_tensorflow/fully_trained/ssd_mobilenet_v1_coco_2018_01_28/ \
-                --run_once=True"}]'
+def run_inference(model_name, models_dir, json_file_name):
+    data = parse_json(args.json_file_name)
 
     try:
-        data = json.loads(parameters)
+        data
```

Comment:
What is the reason for just testing "data" here in the try block, and not testing "data = parse_json(args.json_file_name)"?  The latter is easier to understand, and matches the original code more closely.

Reply:
@crlishka . I was following sindhu's original sequence of lines: replace "parameters" with "data = parse_json(args.json_file_name)", and then when it comes to try block, I realized that look like there I can only put something like "data". Yes, agree with you that your recommendation is better. So I will do this: move the line "data = parse_json(args.json_file_name)" to the position of "data", and delete "data" there. and re-do the test.